src/test/resources/features/99d_employment_qa_smoke.feature

```diff
@@ -0,0 +1,153 @@
+    Then the user should see the correct Other benefits text
+    Then the user should see the correct Assets text
+    Then the user should see the correct Benefits for asset transfers text
+    When the user clicks the back button
+    Then the user should be redirected to the "Employment" page
+    And the user clicks on the expenses link
+    Then the user should be redirected to the "Check your employment expenses" page
+    Then the user should see the correct Amount for business travel and subsistence expenses text
+    Then the user should see the correct Job expenses text
+    Then the user should see the correct Uniforms, work clothes, or tools text
+    Then the user should see the correct Professional fees and subscriptions text
+    Then the user should see the correct Hotel and meal expenses text
+    Then the user should see the correct Other expenses text
+    Then the user should see the correct Vehicle expenses text
+    Then the user should see the correct Mileage allowance relief text
+    When the user clicks the back button
+    Then the user should be redirected to the "Employment" page
+
+
+  Scenario: Returning flow - Agent User with prior UK Employment, Review & Change - Employment Smoke
+    When the user logs into the service with the following details
+      |Redirect url       |/test-only/2022/additional-parameters?ClientNino=AA133742A&ClientMTDID=9876543211  |
+      |Credential Strength|weak                                                                               |
+      |Confidence Level   |200                                                                                |
+      |Affinity Group     |Agent                                                                              |
+      |Enrolment Key 1    |HMRC-MTD-IT                                                                        |
+      |Identifier Name 1  |MTDITID                                                                            |
+      |Identifier Value 1 |9876543211                                                                         |
+      |Enrolment Key 2    |HMRC-AS-AGENT                                                                      |
+      |Identifier Name 2  |AgentReferenceNumber                                                               |
+      |Identifier Value 2 |XARN7654321                                                                        |
+    Then the user should be redirected to the "Update and submit an Income Tax Return" page
+    When the user clicks the continue button
+    Then the user should be redirected to the "Your client’s Income Tax Return" page
+    When the user clicks on the employment link
+    Then the user should be redirected to the "Employment" page
+    When the user clicks on the employment details link
+    Then the user should be redirected to the "Check your client’s employment details" page
+    Then the user should see the correct Employer text
```

Comment:
i dont get why were checking values on the page for the employment data, we havet done this for the other smoke tests